gulpfile.js

```diff
@@ -103,8 +103,11 @@ gulp.task('copy:main.css', function () {
 
     return gulp.src(dirs.src + '/css/main.css')
                .pipe(plugins.header(banner))
-               .pipe(gulp.dest(dirs.dist + '/css'));
-
+                .pipe(plugins.autoprefixer({
+                    browsers: ['last 2 versions', 'ie >= 8'],
```

Comment:
Did some quick tests, and at least, [Opera Mini for iOS 8.2](https://cloud.githubusercontent.com/assets/1223565/6637108/150f5e80-c97f-11e4-8323-7ca863f981f6.png) (with [_Opera Mini mode_](https://cloud.githubusercontent.com/assets/1223565/6637095/e21dd9de-c97e-11e4-8147-823ab3ea6254.png) enabled) and [Opera Mini for Android 4.3](https://cloud.githubusercontent.com/assets/1223565/6637123/4ecc5466-c97f-11e4-9b9b-fc9662789c9f.png), use `(min-resolution: 120dpi)`.